### ISSUE TITLE
A quick getting started alternative without Meta Business App

### ISSUE THREAD

**Issue by jurmy24** [Tanzania-AI-Community/twiga#59]
## Concept
Since the Meta Business App is quite a tedious process to go through (as is seen in the getting_started file), we want an alternative that is really easy to setup and that only requires an LLM API key. 

## Solution concept
One good solution for this could be to run a frontend WhatsApp clone in a browser and interact with the chatbot in that way. It's probably fine to disregard any WhatsApp flows and interactive messages in here and consider it as a playground for testing prompts and RAG features. 

It's probably best to package this using Docker a docker container that automatically creates and spins up a postgres database, a frontend, and runs the fastapi server all at once. 

## Others
Once this is complete we need to update the documentation so that people know how to run it. If we want to test it out we can also see if its possible to run it in GitHub codespaces.

**Comment by jurmy24** [comment]
Note: it might be a good idea to use "volumes" in a docker compose to make it possible to have hot reloading during development from your own computer

**Comment by jurmy24** [comment]
Would also be nice to have postgres setup within the docker container
